Samples/VShadowVolumeShadowCopy/cpp/shadow.cpp

```diff
@@ -689,18 +689,23 @@ int CommandLineParser::MainRoutine(vector<wstring> arguments)
             HRESULT is_supported_result = m_vssClient.IsVolumeSupported(wszVolumePathName, &supported);
 
             const wstring result_phrase = L"\nSupported check is done, bool: %i, NESTED_VOLUME_LIMIT: %i";
+            short trunked = short(is_supported_result);
```

Comment:
Avoid repeated type name in left and right parts of variable declaration. Use `auto` instead.